drivers/spi/spi_ll_stm32.c

```diff
@@ -348,6 +572,14 @@ static int spi_stm32_configure(struct device *dev,
 	ll_func_set_fifo_threshold_8bit(spi);
 #endif
 
+	/* with LL_SPI_FULL_DUPLEX mode, both tx and Rx DMA are on */
```

Comment:
Shouldn't this be under #idef CONFIG_SPI_STM32_DMA?

Reply:
yes, changed as the data->dev_dma_tx and data->dev_dma_rx are related to dma too